noetic/distribution.yaml

```diff
@@ -7220,6 +7220,12 @@ repositories:
       url: https://github.com/rm-controls/rm_controllers.git
       version: master
     status: developed
+  rm_description:
+    doc:
+      type: git
+      url: https://github.com/rm-controls/rm_description.git
```

Comment:
Was `rm_description` moved from `rm-controls/rm_control` to `rm-controls/rm_description`?
Are you still planning to release it from `rm-controls/rm_control` and only want to have a doc entry pointing to `rm-controls/rm_description`?

Reply:
I just want to separate rm_description from rm-controls/rm_control to rm-controls/rm_description. I plan to release it from rm-controlss/rm_description. @ivanpauno

Reply:
@ivanpauno